<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>490</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.500000000000000" starty="1.000000000000000" endx="0.489000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>173</red>
          <green>178</green>
          <blue>208</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>12</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">gibpix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.489, y2:0, stop:0 rgba(173, 178, 208, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTabWidget" name="windows">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.489, y2:0, stop:0 rgba(173, 178, 208, 255), stop:1 rgba(255, 255, 255, 255));</string>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>цвет1</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QSlider" name="red2">
                    <property name="minimumSize">
                     <size>
                      <width>22</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 208, 203);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="green2">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(199, 255, 201);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="blue2">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(192, 221, 255);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>цвет2</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QSlider" name="red1">
                    <property name="cursor">
                     <cursorShape>ArrowCursor</cursorShape>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 208, 203);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="green1">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(199, 255, 201);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="blue1">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(192, 221, 255);</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="tracking">
                     <bool>true</bool>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Page</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="newYearButton">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rainbowButton">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="colorfulQWW">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="button">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Подстроиться под монитор</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Скрыть окно</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
